### PR TITLE
[Self-Hosted] Highlight that the external database name needs to be `gitpod`

### PR DESCRIPTION
**Pull request by lucasvaltl**
## Description
As of right now, the name of a database needs to be `gitpod`. We do not make this clear in our documentation and this has impacted customers. 

## How to test
- sense check changes in files

<a href="https://gitpod-staging.com/#https://github.com/gitpod-io/website/pull/2678"></a>

<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/2678"></a>

### CODE REVIEW

File: gitpod/docs/self-hosted/latest/reference-architecture/_chunks/database.md [hunk 1]
```diff
@@ -10,13 +10,15 @@ Gitpod uses a **relational database management system** to store structural data
 
 In this reference architecture, we use managed MYSQL databases provided by cloud providers.
 
+> Both the database name and the username of the user used to access it need to be `gitpod`.
```
Comment (MrSimonEmms) [inline gitpod/docs/self-hosted/latest/reference-architecture/_chunks/database.md:13]: Only the database name needs to be `gitpod`. The username is configurable as of https://github.com/gitpod-io/gitpod/pull/6171

```suggestion
> The name of the database needs to be `gitpod`.
```

**Review by MrSimonEmms — CHANGES_REQUESTED** [review]
See comment

**Review by MrSimonEmms — APPROVED** [review]
Perfecto

**Comment by adrienthebo** [comment]
Are we certain that the database _instance_ needs to be named gitpod? I know that gitpod uses a MySQL database named `gitpod`, but I don't know that the name of the server running MySQL also needs to be named gitpod.

The [GKE terraform module](https://github.com/gitpod-io/gitpod/blob/main/install/infra/modules/gke/main.tf#L183) indicates that the server name can vary.

Throwing a quick hold on this while I verify.

File: gitpod/docs/self-hosted/latest/reference-architecture/_chunks/database.md [hunk 2]
```diff
@@ -10,13 +10,15 @@ Gitpod uses a **relational database management system** to store structural data
 
 In this reference architecture, we use managed MYSQL databases provided by cloud providers.
 
+> The name of the database needs to be `gitpod`.
```
Comment (adrienthebo) [inline gitpod/docs/self-hosted/latest/reference-architecture/_chunks/database.md:13]: Expanding on my comment on the issue itself, I think we're coupling together the name of the database instance that's running MySQL, and the names of the MySQL databases. The latter definitely needs to be named `gitpod` but that'll be automatically created during the installation of Gitpod itself.

In addition, Gitpod will create a database named something like `gitpod-sessions` - this touches the domain of documenting requirements an making a distinction between requirements and setup procedures.

File: gitpod/docs/self-hosted/latest/reference-architecture/_chunks/database.md [hunk 3]
```diff
@@ -10,13 +10,15 @@ Gitpod uses a **relational database management system** to store structural data
...
+
 <CloudPlatformToggle id="cloud-platform-toggle-database">
 <div slot="gcp">
 
 As a relational database, we create a [Google Cloud SQL instance](https://cloud.google.com/sql) with MySQL 5.7. Use the following commands to create the database instance:
 
 ```bash
-MYSQL_INSTANCE_NAME=gitpod-mysql
+MYSQL_INSTANCE_NAME=gitpod
```
Comment (adrienthebo) [inline gitpod/docs/self-hosted/latest/reference-architecture/_chunks/database.md:21]: 👍

Comment (lucasvaltl) [reply gitpod/docs/self-hosted/latest/reference-architecture/_chunks/database.md:21]: Now that I [understand](https://gitpod.slack.com/archives/C01KLC56NP7/p1662044900851759?thread_ts=1662028843.913359&cid=C01KLC56NP7) this better, I like the before better. Will change it back: 
```suggestion
MYSQL_INSTANCE_NAME=gitpod-mysql
```

File: gitpod/docs/self-hosted/latest/reference-architecture/_chunks/database.md (hunk 2 above)
Comment (adrienthebo) [reply gitpod/docs/self-hosted/latest/reference-architecture/_chunks/database.md:13]: LGTM!

**Review by adrienthebo — APPROVED** [review]

**Pull request merged**